cmd/verify.go

```diff
@@ -0,0 +1,52 @@
+package cmd
```

Comment:
this function was part of main.go, I just moved it in here. No change to the code itself